mixnode/src/commands/run.rs

```diff
@@ -172,17 +175,15 @@ pub fn execute(matches: &ArgMatches) {
     Sphinx key: {}
     Host: {}
     Layer: {}
-    Location: [physical location of your node's server]
+    Location: {}
```

Comment:
Isn't location going away from bonding hence this shouldn't be shown anymore?

Reply:
You're right, it doesn't make sense anymore to have it here. I'll remove it.